test/spec/modules/parrableIdSystem_spec.js

```diff
@@ -242,6 +247,86 @@ describe('Parrable ID System', function() {
         })
       });
     });
+
+    describe('third party cookie support', function () {
+      let logErrorStub;
+      let callbackSpy = sinon.spy();
+
+      beforeEach(function() {
+        logErrorStub = sinon.stub(utils, 'logError');
+      });
+
+      afterEach(function () {
+        callbackSpy.resetHistory();
+        removeParrableCookie();
+      });
+
+      afterEach(function() {
+        logErrorStub.restore();
+      });
+
+      describe('when getting tpcSupport from XHR response', function () {
```

Comment:
I think we need a parallel block
```js
describe('when reading tpcSupport status from cookie');
```

and it tests sending the tpc support to the backend, and then when the tpcUntil expires it should test suppressing the status from the cookie.